packages/shared/__mocks__/svgrMock.ts

```diff
@@ -1 +1 @@
-export default 'div';
+export default 'svg';
```

Comment:
Just so that tests are more silent and doesn't complain about a `div` inside `x` or whatever.